platform-configs/include/pcengines.robot

```diff
@@ -6,7 +6,9 @@ Resource    ../../keys.robot
 *** Variables ***
 ${INITIAL_DUT_CONNECTION_METHOD}=           Telnet
 ${DUT_CONNECTION_METHOD}=                   ${INITIAL_DUT_CONNECTION_METHOD}
-${PAYLOAD}=                                 tianocore
+${BIOS_LIB}=                                seabios
+${FW_STRING}=                               F10
+${SEABIOS_BOOT_DEVICE}=                     4
```

Comment:
Hmm. We also have UEFI variant of the firmware. How will we deal with it? IS there a separate PC Engines config for UEFI variant?